!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  blue_h: 100
  blue_s: 247
  blue_v: 205
  blur_kernel_size: 3
  expected_sphero_radius: 25
  fwd_hue_max: 255
  fwd_hue_min: 0
  fwd_sat_max: 255
  fwd_sat_min: 0
  fwd_val_max: 255
  fwd_val_min: 220
  green_h: 91
  green_s: 155
  green_v: 200
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      blue_h: 100
      blue_s: 247
      blue_v: 205
      blur_kernel_size: 3
      expected_sphero_radius: 25
      fwd_hue_max: 255
      fwd_hue_min: 0
      fwd_sat_max: 255
      fwd_sat_min: 0
      fwd_val_max: 255
      fwd_val_min: 220
      green_h: 91
      green_s: 155
      green_v: 200
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      hue_max: 5
      hue_min: 5
      id: 0
      lower_threshold: 233
      max_contour_area: 750
      min_contour_area: 100
      morph_rect_size: 3
      name: Default
      nine_sharpen_kernel: true
      orientation_covariance: 0.1
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      position_covariance: 3.0
      red_h: 175
      red_s: 162
      red_v: 200
      sat_max: 15
      sat_min: 15
      show_pipeline: true
      state: true
      type: ''
      upper_threshold: 255
      val_max: 50
      val_min: 50
    state: []
  hue_max: 5
  hue_min: 5
  lower_threshold: 233
  max_contour_area: 750
  min_contour_area: 100
  morph_rect_size: 3
  nine_sharpen_kernel: true
  orientation_covariance: 0.1
  position_covariance: 3.0
  red_h: 175
  red_s: 162
  red_v: 200
  sat_max: 15
  sat_min: 15
  show_pipeline: true
  upper_threshold: 255
  val_max: 50
  val_min: 50
state: []
